digiroad2-api-oth/src/main/scala/fi/liikennevirasto/digiroad2/Digiroad2Api.scala

```diff
@@ -361,11 +396,22 @@ class Digiroad2Api(val roadLinkService: RoadLinkService,
         "propertyData" -> stop.propertyData,
         "municipalityCode" -> massTransitStopReturned._3)
     }
+  }
```

Comment:
Is any bus stop being returned here?

Reply:
No. Because of an accidental remove of some code this is not doing what was supposed too. I will be reverted